index.md

```diff
@@ -2,14 +2,14 @@
 layout: lesson
 ---
 
-The core tools (shell, version control, etc) and how you could make
-your research more reproducible having gained improved knowledge of
-them.
+Welcome to the Reproducible Basics Module!
+
+This module will show how to use several core tools: shell, version control, etc. It will explain how proficiency in the use of these tools promotes reproducibility in your research.
```

Comment:
I think that there was some agreement to have here only a paragraph describing what the module is about, for incorporation into moodle or something.  So there should be no opening phrase or "This module will show ...". So may be just keep it here framed as it was for now